doc/tasks/git_commit_message.md

```diff
@@ -10,8 +10,12 @@ parameters:
         git_commit_message:
             allow_empty_message: false
             enforce_capitalized_subject: true
+            enforce_no_subject_punctuations: false
             enforce_no_subject_trailing_period: true
             enforce_single_lined_subject: true
+            enforce_type_scope_conventions: false
+            types: []
```

Comment:
Invalid indentation

Reply:
I'm a bit confused here, These should had be removed after the force push. I need to investigate this.

Reply:
Nevermind, it was just overlooked when force pushing the new proposal. A fix will be pushed soon.

Reply:
This should resolve this conversation https://github.com/phpro/grumphp/pull/541/commits/aa6d1ee5ab70d8408b731d2b645f76291f6687ce